src/com/ubcsolar/ui/CustomDisplayMap.java

```diff
@@ -391,10 +533,20 @@ public void refreshMap() {
 			}
 		}
 
-		if (this.showSpeeds && speeds != null) {
+		if (this.showSpeeds && speeds != null && accelBreadcrumbs != null && deccelBreadcrumbs != null) {
 			for (MapMarker m : speeds) {
 				this.addMapMarker(m);
 			}
+			for (MapPolygon chunk : this.accelBreadcrumbs) {
+				((MapObjectImpl) chunk).setColor(Color.green);
```

Comment:
I was testing to see if there are different shades of green but they are the same. I will change it back next commit for consistency